tests/e2e/utils/src/components.js

```diff
@@ -322,6 +322,49 @@ const createGroupedProduct = async (groupedProduct = defaultGroupedProduct) => {
 	return id;
 };
 
+/**
+ * Use the API to create an order with the provided details.
+ *
+ * @param {object} orderOptions
+ * @returns {Promise<number>} ID of the created order.
+ */
+const createOrder = async ( orderOptions = {} ) => {
```

Comment:
We should use `orderRESTRepository`.

The function docs should list what order properties the function supports since it doesn't support all properties.

Reply:
Good call! I updated this and also simplified the logic a bit to use the spread operator instead of spread out `if` statements.